src/Microsoft.TestPlatform.Extensions.BlameDataCollector/Resources/Resources.resx

```diff
@@ -133,7 +133,7 @@
     <value>The specified inactivity time of {0} minute/s has elapsed. Collecting a dump and killing the test host process.</value>
   </data>
   <data name="NotGeneratingSequenceFile" xml:space="preserve">
-    <value>All tests finished running, Sequence file will not be generated.</value>
+    <value>All tests finished running, Sequence file will not be generated</value>
```

Comment:
removed the . because blame collector will add it and there were 2. Like this: `will not be generated..`

Reply:
Maybe mention that on <comment>, that way translators will have context and warning for not adding the dot too.

Reply:
I don't think they try to add any punctuation, but IDK. Did you get a message from them about that anytime?

Reply:
Nope. It's just a hunch.

Reply:
Done.